src/editorial-source-components/InputHeading.tsx

```diff
@@ -20,7 +20,7 @@ const Errors = ({ errors }: { errors: string[] }) =>
   !errors.length ? null : <Error>{errors.join(", ")}</Error>;
 
 type Props = {
-  label: string;
+  label: React.ReactNode;
```

Comment:
question: we are changing to `React.ReactNode` whilst still passing in a string from the various forms?

Im assuming this is being used to allow for a potential `null` or `undefined` prop value?
Just hoping Im on the right lines https://reactnative.dev/docs/react-node

Reply:
It's true that this allows a null value, a string value, or a React element! Ultimately, I think it's just useful to provide a label 'slot' that the user can fill with whatever they want – text, additional content, etc.

It's required in this case to allow us to pass our `Copy from alt text` button in as part of the label content.